src/test/unit/test_tracer.py

```diff
@@ -51,9 +53,25 @@ def lambda_test_function(event, context):
     assert first_send[0]["maxFinishTime"]
 
 
+@pytest.mark.parametrize("token", ["t_", "", "10faa5e13e7844aaa1234"])
+def test_lambda_wrapper_validate_token_format(context, capsys, token):
+    """
+    This test checks that the token has a valid format with only prefix
+    """
+
+    @lumigo_tracer(token=token)
+    def lambda_test_function(event, context):
+        pass
+
+    lambda_test_function({}, context)
+    captured = capsys.readouterr()
+    expected = "Lumigo Warning: Invalid token used, copy your token from Settings → Tracing from Lumigo’s platform\n"
+    assert captured[0] == expected
```

Comment:
please also check that the tracer haven't send any span to Lumigo

Reply:
what is span?

Reply:
`reporter_mock.call_count == 2`
And use the fixture `reporter_mock`